rasa_core_sdk/forms.py

```diff
@@ -311,16 +311,32 @@ def extract_requested_slot(
                     )
                     return {slot_to_fill: value}
 
-        logger.debug("Failed to extract requested slot '{}'" "".format(slot_to_fill))
+        logger.debug("Failed to extract requested slot '{}'".format(slot_to_fill))
         return {}
 
+    # def validate_<slot_name>(self, value, dispatcher, tracker, domain):
```

Comment:
@tmbo we want to provide generic example with all the in/out types here (like a template method), but users cannot directly override it in subclass, since the name will be different. Is it a good way to showcase it?

Reply:
not sure I fully understand, so it would be in here as a comment?

Reply:
yeah, I think that's the idea (not sure if you noticed but there's a whole function here, not just the first line). Another idea is we could write a full (non-commented) function for something titled `validate_example_slot()` and then the description would be something like "an example custom validation function for validating a slot named "example_slot". You can use this template to create custom validation functions for your slots", or something like that

Reply:
Ok in that case, how about the following: use the goolge python style `Example` section in the `validate` methods description to show how it should be used (this will make sure the syntax is correct, while this comment won't)